src/oslinfo/oslinfo.cpp

```diff
@@ -235,6 +244,7 @@ main (int argc, char *argv[])
                 "-h", &help, "Print help message",
                 "--help", &help, "",
                 "-v", &verbose, "Verbose",
+                "--bench", &benchmark, "Benchmark shader loading time",
```

Comment:
That's totally valid. I can change to `--runstats`.

How hard would it be to doctor the benchmarking to also simulate reading the shader into the renderer so we can benchmark that more easily?